src/components/pages/home/index.js

```diff
@@ -53,9 +45,10 @@ const Home = () => (
 
     <LinkContainer hasColumns>
       <UploadLink to='/upload'>Upload PDFs of patient data</UploadLink>
-      <SearchLink to='/enter'>Or type in patient details</SearchLink>
-    </LinkContainer>
+      <SearchLink to='/enter'><Keyboard fill={colors.primary} width={30}/>Or type in patient details</SearchLink>
 
+      <SearchLink isSolid to='/enter'><Keyboard fill={colors.white} width={30}/>Or type in patient details</SearchLink>
+    </LinkContainer>
```

Comment:
this is the same sentence 
do we need both of them?

Reply:
I added both buttons to serve as an example of how to pass in `isSolid` to give an icon button with a purple background. Will remove. Same reasoning behind leaving the route in with the icons on the card.

@susanX @wright1 @othman-shamla would you like me to create an issue for how to pass in props to style buttons and icons, or can you figure this out ok?

Reply:
@MohammedYehia changes made